zeroconf/const.py

```diff
@@ -34,6 +34,12 @@
 _DUPLICATE_QUESTION_INTERVAL = _BROWSER_TIME - 1  # ms
 _BROWSER_BACKOFF_LIMIT = 3600  # s
 _CACHE_CLEANUP_INTERVAL = 10000  # ms
+_LOADED_SYSTEM_TIMEOUT = 10  # s
```

Comment:
10 seconds is a bit large of a timeout, its only unused in a cases where we wrap other timeouts so even a larger number would probably be fine as the only time we expect this timeout to happen in a production circumstance is when someone is running a `ServiceInfo.request()` in another thread and another thread calls `Zeroconf.close()` at just the right moment that the future is never completed.